packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -301,8 +301,10 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
       index = options.length - 1;
     }
     let stepCounter = 0;
-    // If current index is a header or divider, increment by step
-    while (options[index].itemType === DropdownMenuItemType.Header || options[index].itemType === DropdownMenuItemType.Divider) {
+    // If current index is a header or divider, or disabled, increment by step
+    while (options[index].itemType === DropdownMenuItemType.Header ||
+      options[index].itemType === DropdownMenuItemType.Divider ||
+      options[index].disabled) {
```

Comment:
I believe that disabled items should still be selectable for accessibility reasons. If someone isn't sighted, how would they know there were disabled items?

Reply:
Good catch @joschect. @jspurlin for fyi!

Reply:
Not selectable, but focusable?